mixer/cmd/server/cmd/server.go

```diff
@@ -15,426 +15,103 @@
 package cmd
 
 import (
-	"bytes"
-	"crypto/tls"
-	"crypto/x509"
 	_ "expvar" // For /debug/vars registration. Note: temporary, NOT for general use
-	"fmt"
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-	_ "net/http/pprof" // For profiling / performance investigations
-	"strings"
-	"time"
 
-	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
-	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
-	ot "github.com/opentracing/opentracing-go"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/cobra"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials"
 
-	mixerpb "istio.io/api/mixer/v1"
 	"istio.io/istio/mixer/cmd/shared"
 	adptr "istio.io/istio/mixer/pkg/adapter"
```

Comment:
any reason to keep this as `adptr` (instead of the base `adapter`) at this point?

Reply:
Removed.